frontend/src/pages/DashboardPage/Demographics/DemographicsSelector.jsx

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+
+const DemographicsSelector = ({
```

Comment:
Is the dashboard functional with the changes you made?

Reply:
Yea its working